dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/CollectionRestRepository.java

```diff
@@ -165,7 +168,8 @@ public CollectionRest findOne(Context context, UUID id) {
 
     @SearchRestMethod(name = "findAuthorizedByCommunity")
     public Page<CollectionRest> findAuthorizedByCommunity(
-        @Parameter(value = "uuid", required = true) UUID communityUuid, Pageable pageable) {
+        @Parameter(value = "uuid", required = true) UUID communityUuid, Pageable pageable,
+        @Parameter(value = "query") String q) {
```

Comment:
What is the use case for this new "query" parameter ? Because it isn't in the rest contract: https://github.com/DSpace/Rest7Contract/blob/733d81061f1dbcd3049ed0d80130746418a4b3a1/collections.md#findauthorizedbycommunity

Reply:
thanks to point to the rest contract. Indeed we need to create a PR to document this change. The need for a query parameter was noted and agreed in the discussion about how to address the performance issue https://wiki.lyrasis.org/display/DSPACE/Collection+dropdown+in+submission

Reply:
https://github.com/DSpace/Rest7Contract/pull/127